components/NotFound.vue

```diff
@@ -0,0 +1,20 @@
+<template>
```

Comment:
> Warning: Though this file is placed in the layouts folder, it should be treated as a page.

From [here](https://nuxtjs.org/guide/views/#error-page)

Move this to layouts?

Reply:
No worry, I just finished it to use their native error call ;)